dataprofiler/profilers/utils.py

```diff
@@ -332,7 +332,7 @@ def biased_skew(df_series: Series) -> float:
     """
     n = len(df_series)
     if n < 1:
-        return np.nan
+        return float(np.nan)
```

Comment:
np.nan is specific, I think we should maintain the np.nan type if possible. is np.nan not accepted or do we need to fix to allow?

Reply:
Oh this cast isn't needed once I added numpy as a dependency in the mypy pre-commit hook. I'll remove it now.

Reply:
great